frontend/server/src/CreateUserParams.php

```diff
@@ -57,7 +57,19 @@ class CreateUserParams {
     public $recaptcha = null;
 
     /**
-     * @param array{username?: string, name?: string, email?: string, password?: string, scholar_degree?: string, is_private?: string, gender?: string, recaptcha?: string} $params
+     * @readonly
+     * @var int
+     */
+    public $birthDate;
```

Comment:
It could be temporary null, until we fix the third party login method
```suggestion
     * @var int|null
     */
    public $birthDate = null;
```